ddtrace/internal/writer/writer.py

```diff
@@ -441,6 +441,7 @@ def _flush_single_payload(
                     n_traces,
                     self._intake_endpoint(client),
                     self.RETRY_ATTEMPTS,
+                    exc_info=True,
```

Comment:
this should be fine here, since this is running in a thread this should not include many/any user app frames in the stack trace.

Reply:
if you are concerned, we mostly just need the exception type and message, so adding `type(e), str(e)` to the log message could be enough as well.

I don't think the stack actually helps us here since we know why this got triggered... right?